setup/MSBuild.vsmanproj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <Import Project="$(MicroBuildDir)MicroBuild.Core.props" />
+
+  <PropertyGroup>
+    <!-- Define properties that drive the manifest creation here. -->
+    <FinalizeManifest>true</FinalizeManifest>
+    <FinalizeSkipLayout>true</FinalizeSkipLayout>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutDir>$(BaseOutputPath)setup\</OutDir>
+    <OutputPath>$(OutputPath)setup\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="MSBuild.swixproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <MergeManifest Include="$(BaseOutputPath)setup\*.json" />
```

Comment:
Because these files are produced by `MSBuild.swixproj`, this wildcard is perilous. Since it's outside a target, the wildcard is eagerly expanded in pass 3 of this project's evaluation, before any targets (including the target that deals with `ProjectReference`s) are built. So it's not guaranteed to contain anything (on a clean build starting with this project).

I would recommend moving this inside a target with `DependsOn="ResolveProjectReferences"` and `BeforeTargets=` whatever consumes this item. Then it would always be up to date.

Since this is built as a separate step after the other project in the build definition, this is isn't a must-fix here.